Modules/_ctypes/callproc.c

```diff
@@ -1443,40 +1439,31 @@ copy_com_pointer(PyObject *self, PyObject *args)
 }
 #else
 
-#ifdef HAVE_DYLD_SHARED_CACHE_CONTAINS_PATH
-static PyObject *py_dyld_shared_cache_contains_path(PyObject *self, PyObject *args)
-{
-     PyObject *name, *name2;
-     char *name_str;
-
-     if (__builtin_available(macOS 11.0, iOS 14.0, tvOS 14.0, watchOS 7.0, *)) {
-         int r;
-
-         if (!PyArg_ParseTuple(args, "O", &name))
-             return NULL;
-
-         if (name == Py_None)
-             Py_RETURN_FALSE;
-
-         if (PyUnicode_FSConverter(name, &name2) == 0)
-             return NULL;
-         name_str = PyBytes_AS_STRING(name2);
+#ifndef MS_WIN32
+static PyObject *py_shared_library_is_loadable(PyObject *self, PyObject *args) {
+    PyObject *name, *name2;
+    char *name_str;
+    void* handle;
 
-         r = _dyld_shared_cache_contains_path(name_str);
-         Py_DECREF(name2);
+    if (!PyArg_ParseTuple(args, "O", &name))
+        return NULL;
 
-         if (r) {
-             Py_RETURN_TRUE;
-         } else {
-             Py_RETURN_FALSE;
-         }
+    if (name == Py_None)
+        Py_RETURN_FALSE;
 
-     } else {
-         PyErr_SetString(PyExc_NotImplementedError, "_dyld_shared_cache_contains_path symbol is missing");
-         return NULL;
-     }
+    if (PyUnicode_FSConverter(name, &name2) == 0)
+        return NULL;
```

Comment:
PEP 7: _"All new C code requires braces"_

This also applies when modifying existing code, so I'd add braces for those `if`'s.